web/src/types/user.type.ts

```diff
@@ -0,0 +1,5 @@
+export interface User {
+  id: string;
+  name: string;
+  image: string;
```

Comment:
I would prefer `profileImageURL` here, assuming that is in fact what this is

Reply:
I was trying to be consistent with the names we have for the columns on the db